src/rez/rezconfig.py

```diff
@@ -1212,46 +1212,7 @@
 # common to all plugins of that type.
 
 plugins = {
```

Comment:
I'm curious to know what would happen if we removed this entirely from here.

Reply:
Running `rez config plugins` is how I've been verifying that my changes are working. Removing this line and attempting to run `rez config plugins` causes an error:

```
Traceback (most recent call last):
  File "e:\code\rez\venv\Lib\site-packages\rez\cli\config.py", line 53, in command
    data = data[key]
           ~~~~^^^^^
KeyError: 'plugins'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "E:\Code\rez\venv\Scripts\rez\rez.exe\__main__.py", line 7, in <module>
  File "e:\code\rez\venv\Lib\site-packages\rez\cli\_entry_points.py", line 69, in run_rez
    return run()
           ^^^^^
  File "e:\code\rez\venv\Lib\site-packages\rez\cli\_main.py", line 189, in run
    returncode = run_cmd()
                 ^^^^^^^^^
  File "e:\code\rez\venv\Lib\site-packages\rez\cli\_main.py", line 181, in run_cmd
    return func(opts, opts.parser, extra_arg_groups)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "e:\code\rez\venv\Lib\site-packages\rez\cli\config.py", line 55, in command
    raise ValueError("no such setting: %r" % opts.FIELD)
ValueError: no such setting: 'plugins'
```